src/hotspot/cpu/aarch64/stubGenerator_aarch64.cpp

```diff
@@ -1333,7 +1333,7 @@ class StubGenerator: public StubCodeGenerator {
   //   disjoint_int_copy_entry is set to the no-overlap entry point
   //   used by generate_conjoint_int_oop_copy().
   //
-  address generate_disjoint_copy(size_t size, bool aligned, bool is_oop, address *entry,
+  address generate_disjoint_copy(int size, bool aligned, bool is_oop, address *entry,
```

Comment:
size_t is correct here.

Reply:
I switched back: https://github.com/openjdk/jdk11u-dev/pull/280/commits/09587d5e38d8bdd2ad70a585ace7c156c9be1418
This way, however, I had to add casts to the two call sites where `size` is used, to avoid a `size_t` to `int` truncation warning ([C4267](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-3-c4267?view=msvc-170)). Both calls expect integers. Making `size` an `int` here avoided both the warning and the casts.